src/ResourceManager/DeviceProvisioningServices/AzureRM.DeviceProvisioningServices.psd1

```diff
@@ -0,0 +1,134 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.DeviceProvisioningServices'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 2/21/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '1.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'a9f5b86f-63be-4e7a-8e16-c8c7c36cae40'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - Iot Device provisioning service cmdlets for Azure Resource Manager'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '3.0'
```

Comment:
@anusapan for //Build, we are looking to bump the minimum version of PowerShell required to be 5.0, so this should be the value used

Reply:
Done